lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -144,6 +139,11 @@ export class PopupHandler extends InteractionHandler {
             if (popupWindow.focus) {
                 popupWindow.focus();
             }
+            window.onbeforeunload = (): void => {
```

Comment:
Use `addEventListener` so that you don't blow away events registered by other scripts.